enterprise_subsidy/apps/subsidy/models.py

```diff
@@ -324,11 +320,10 @@ def _create_redemption(self, learner_id, content_key, subsidy_access_policy_uuid
         )
 
         try:
-            reference_id = self.enterprise_client.enroll(learner_id, content_key, ledger_transaction)
+            enterprise_fulfillment_uuid = self.enterprise_client.enroll(learner_id, content_key, ledger_transaction)
             self.commit_transaction(
                 ledger_transaction,
-                reference_id=reference_id,
-                reference_type=OCM_ENROLLMENT_REFERENCE_TYPE,
+                fulfillment_identifier=enterprise_fulfillment_uuid,
```

Comment:
Can you add a TODO comment here to pass an external_reference when we add GEAG support?